oauth2/oauth2-resource-server/src/test/java/org/springframework/security/oauth2/server/resource/web/server/authentication/ServerBearerTokenAuthenticationConverterTests.java

```diff
@@ -217,6 +217,19 @@ void resolveWhenQueryParameterHasMultipleAccessTokensThenOAuth2AuthenticationExc
 
 	}
 
+	@Test
+	public void resolveWhenQueryParameterIsPresentAndEmptyStringThenTokenIsNotResolved() {
```

Comment:
I believe this is the same test as `resolveWhenQueryParameterIsEmptyAndSupportedThenOAuth2AuthenticationException()`? I think this test isn't needed?

Reply:
Hi @sjohnr.
This test was deemed unnecessary and was removed. thank you.